lib/runtime-c-api/src/lib.rs

```diff
@@ -17,23 +17,23 @@ use wasmer_runtime_core::module::{ExportIndex, ImportName};
 use wasmer_runtime_core::types::{ElementType, FuncSig, MemoryDescriptor, TableDescriptor, Type};
 use wasmer_runtime_core::units::{Bytes, Pages};
 
-#[allow(non_camel_case_types)]
-pub struct wasmer_module_t();
+#[repr(C)]
+pub struct wasmer_module_t;
```

Comment:
I've removed `()` at the end of the structure declaration. I don't think they are required.

Reply:
👍